### PR TITLE
fix: clean code from static names

### PR DESCRIPTION
**Pull request by isaac-fibonacci**
## What does this MR do?

Cleane Code from Static Names

## Check list  

- [X] All Merge Request pipelines should be passed.
- [X] Make sure all the changes are necessary and correct.
- [X] Make sure no needed change missing (all necessary changes are made).
- [X] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [ ] All necessary unit tests are written.
- [X] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -4,6 +4,10 @@
 NODE_ENV=development
 # ---------------------------------------------------------------------
 
+# Product Name
+DEX_NAME="DEX"
```
Comment (WilliamHawkingStephen) [inline .env.example:8]: - Because someone may not know how many projects we have and with what name, it might be better to have a place for this topic

Comment (isaac-fibonacci) [reply .env.example:8]: We use this parameter wherever we have necessary strings that include the name of the project

**Review by GreatGaryCooper — APPROVED** [review]

**Review by franken5tein — APPROVED** [review]

**Comment by GreatGaryCooper** [comment]
@isaac-fibonacci Please fix the conflicts to merge this PR, it's been a week since there is no activity on it.
Thanks 🙏🏻 🌹

**Comment by GreatGaryCooper** [comment]
Hey @isaac-fibonacci, just wanted to do a follow up on this PR.
Thanks 🙏🏻

File: package.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
   "name": "ts-version",
   "version": "1.0.0",
   "main": "index.js",
-  "author": "Dexpresso Tech Department",
+  "author": "DEX Tech Department",
   "scripts": {
     "build": "tsc",
     "start:dev": "ts-node-dev --files -r tsconfig-paths/register src/index.ts",
     "start:prod": "node -r module-alias/register dist/index.js",
-    "hypersolo": "nodemon --watch 'src/**' --ext 'ts,json' --exec 'yarn build && echo 'Compiled!' && yarn start:prod'",
```
Comment (GreatGaryCooper) [inline package.json:10]: By the way, please remove the upper `start:dev` command and change `start:up` to `start:dev`.
Thanks 🌹

**Pull request closed without merging**